src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -39,8 +39,8 @@
 
     public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
     public static final String NAME_DESC_BOB = " " + PREFIX_NAME + VALID_NAME_BOB;
-    public static final String PHONE_DESC_AMY = " " + PREFIX_PHONE + VALID_PHONE_AMY;
-    public static final String PHONE_DESC_BOB = " " + PREFIX_PHONE + VALID_PHONE_BOB;
+    public static final String PHONE_DESC_AMY = " " + PREFIX_RATING + VALID_PHONE_AMY;
+    public static final String PHONE_DESC_BOB = " " + PREFIX_RATING + VALID_PHONE_BOB;
```

Comment:
VALID_PHONE_AMY and VALID_PHONE_BOB needs to be changed to a valid rating. 
Fixes ExitCommandTest